source/atlas-data-lake-testing/tests/listDatabases.yml

```diff
@@ -1,13 +1,15 @@
 tests:
   -
-    description: "ListDatabaseNames succeeds"
+    description: "ListDatabases succeeds"
     operations:
       -
-        name: listDatabaseNames
+        name: listDatabases
         object: client
     expectations:
       -
         command_started_event:
+          command_name: "listDatabases"
           command:
             listDatabases: 1
+            $db: "admin"
```

Comment:
Likewise, I suggest using `database_name` instead of the OP_MSG `$db` field since we already have that assertion available to us.

```yaml
command_name: "listCollections"
database_name: "admin"
command:
```

Reply:
Updated